client/components/infinite-list/index.jsx

```diff
@@ -223,7 +226,7 @@ export default createReactClass( {
 	scrollChecks() {
 		// isMounted is necessary to prevent running this before it is mounted,
 		// which could be triggered by data-observe mixin.
```

Comment:
Belated drive-by -- is this even still necessary? I don't see `data-observe` being used by this component anymore...